### ISSUE TITLE
Can't save articles behind paywall 😱

### ISSUE THREAD

**Issue by nicosomb** [wallabag/wallabag#4225]
When I save a Mediapart article behind the paywall, I have this error on wallabag.it: 

```
[2019-12-10 09:59:29] app.ERROR: Error: Decrypt fail: Integrity check failed. when sending request: GET https://www.mediapart.fr/journal/international/091219/le-trouble-pactole-qatari-de-l-agent-de-javier-pastore 1.1 {"request":"[object] (GuzzleHttp\\Psr7\\Request: {})","exception":"[object] (Http\\Client\\Exception\\RequestException(code: 0): Decrypt fail: Integrity check failed. at /var/www/wallabag.it/app2019/vendor/php-http/guzzle5-adapter/src/Client.php:132, GuzzleHttp\\Exception\\RequestException(code: 0): Decrypt fail: Integrity check failed. at /var/www/wallabag.it/app2019/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:51, RuntimeException(code: 0): Decrypt fail: Integrity check failed. at /var/www/wallabag.it/app2019/src/Wallabag/CoreBundle/Helper/CryptoProxy.php:61)","milliseconds":1} []
```

@j0k3r any idea?

**Comment by nicosomb** [comment]
I close this issue. I lost the key during the servers transfer.

**Comment by lizyn** [comment]
I have a related issue, and I am not sure if I should open another issue for it.

The whole point is, I think we should make the debug info about paywall more specific.

Last month, I tried to add paywall support for a  site. Unfortunately, I failed at last (turns out the login info was sent through json), and deleted related site-config. But I forgot to delete the credential as well, or I didn't feel the need to. 

Then a few days later, I found out that wallabag started to fail parsing that same site. I thought it was due to a site upgrade, since I haven't bag that site for a while, not including the paywall test chance. 

So I began a long road toward writing "the right site-config" and enabled graby debug (should have enabled it earlier). As a normal user who doesn't code, I took the following log as describing some failure in web data transfer...

`graby.WARNING: Request throw exception (with no response): Decrypt fail: Integrity check failed. {"error_message":"Decrypt fail: Integrity check failed."} []`

Not until seeing this issue of yours did I realize that I hadn't got my site credentials deleted (I was thinking of saving it for future site-config test). So I suggest we do one of these:

1. do not try to decrypt without a site-config where paywall patterns are configured
2. make the log more specific, e.g. adding things like "credential exists but no corresponding fields specified in site-config"
